Flexa-Agents/large-deposit-withdrawal/README.md

```diff
@@ -0,0 +1,28 @@
+# Large Deposit Withdrawal Agent
+
+## Description
+
+This agent detects deposits and withdrawals in Flexa staking contract with an amount that exceeds 1M USD.
+
+> Amp token price is obtained using chainlink data feeds.
+
+## Supported Chains
+
+- Ethereum
+
+## Alerts
+
+- FLEXA-1
+  - Fired when a transaction deposits/ withdraws a large amount to Flexa Staking contract.
```

Comment:
```suggestion
  - Fired when a transaction deposits/withdraws a large amount to/from Flexa Staking contract.
```
Suggestion to add "from" since withdrawals can take place as well.